device-types/Nokia/7210-SAS-M-24F-2XFP.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-M 24F 2XFP
+slug: nokia-7210-sas-m-24f-2xfp
+part_number: 3HE05828AAAB01
+u_height: 2
+is_full_depth: false
```

Comment:
Please add the `airflow` attribute if you have it available

Reply:
Added